frame/evm/src/lib.rs

```diff
@@ -378,6 +390,76 @@ decl_module! {
 
 			ret.map_err(Into::into)
 		}
+
+		/// Issue an EVM create2 operation.
+		#[weight = WeightForCallCreate]
```

Comment:
You can use `weight::FunctionOf` here to make your life easier :D

Reply:
Fixed!